apps/next/next.config.js

```diff
@@ -76,6 +76,7 @@ const nextConfig = {
     "react-native-reanimated-carousel",
   ],
   webpack: (config, options) => {
+    config.resolve.fallback = { fs: false, net: false, tls: false };
```

Comment:
this is recommended in latest rainbowkit as it doesn't polyfill these deps anymore

Reply:
This is not save enough